ontoma/cli.py

```diff
@@ -10,8 +11,15 @@
 logger = logging.getLogger(__name__)
 
 
-def get_version():
-    return open('../VERSION').read().strip()
+def get_version() -> str:
+    """
+    It reads the VERSION file in the same directory as the script and returns its contents
+
+    Returns:
+      The version number of the script.
+    """
+    script_path = os.path.dirname(__file__)
+    return open(f'{script_path}/../VERSION').read().strip()
```

Comment:
Thank you for fixing this! I noticed that when calling `ontoma --version`, it doesn't add a newline. While we're at it, could you please fix this as well in the `cli.py`?

![image](https://user-images.githubusercontent.com/10669118/215465884-9fdd9b17-3baf-4529-aa49-93f0cff3f179.png)